slash/utils/suite_files.py

```diff
@@ -20,7 +20,7 @@ def iter_suite_file_paths(suite_files):
                 path = suite_entry.path
 
                 if not os.path.isabs(path):
-                    path = os.path.abspath(os.path.join(dirname, path))
+                    path = os.path.relpath(os.path.join(dirname, path))
```

Comment:
I'm not sure I understand this change... On which cases is this `if` going to get entered, and have a different value for `path` afterwards?

If I pass `./bla`, relpath is going to still return `./bla`...

Reply:
The relative path in the suite-file will be relative path of the test script with respect to the suite-file path.
when the suite file path is in different path than the path where slash is invoked.
then path of test collected would be relative path of the script from where slash is invoked.
For example
When slash is invoked in same path as the sample_test_repo

![image](https://user-images.githubusercontent.com/18627674/88426189-befdb600-cdbe-11ea-9f4c-f595f4e2c47e.png)

```
slash run -f suites\module_1\suite_file.txt
```
or 
```
slash list -f suites\module_1\suite_file.txt
```
then the path of the tests collected will be relative to sample_test_repo.
tests\feature_one\test_sub_feature1.py

Reply:
@vmalloc do you have any more questions regarding this?

Reply:
I see. @ayalash @yaelmi3 do you have any objection to this change?

Reply:
@vmalloc : LGTM